sections/docker.zsh

```diff
@@ -24,8 +24,11 @@ spaceship_docker() {
 
   spaceship::exists docker || return
 
+  # COMPOSE_FILE may have several filenames separated by colon, test at least the first one
+  COMPOSE_FILE_SPLIT=("${(@s/:/)COMPOSE_FILE}")
```

Comment:
The comment may refer to the official documentation : https://docs.docker.com/compose/reference/envvars/

Also notice the separator can be overridden by `COMPOSE_FILE_SEPARATOR`.

Reply:
done